packages/ui/src/components/Button/button.style.ts

```diff
@@ -59,10 +60,24 @@ export const ButtonStyles = styled.div`
     padding: 0px 10px;
   }
   .medium {
-    height: 32px;
-    padding: 6px 12px;
     min-width: 70px;
+    height: 32px;
     line-height: 0;
+    padding: 6px 12px;
     font-size: 0.875rem;
   }
+  .large {
+    height: 50px;
+    span {
+      position: absolute;
+      display: flex;
+      -webkit-box-align: center;
+      align-items: center;
+      inset: 0px auto 0px 22px;
+      z-index: 1;
```

Comment:
I would rework this. Try to avoid absolute it wont scale well for us. Imagine the case that you have a very long text.

Reply:
hi dear @paschalidi 
I was trying to find another solution until I checked what they did. 
In this case if we have a very long text the icon would still be separated 22px from the left margin and the auto from (`inset: 0px auto 0px 22px;`) will keep it separated from the text if there is space. Even if the text is super long if the button has space the icon would be in the same place.
I will keep trying to find a better solution. Could you please give me another example in which this way wouldn't work? I am just asking because I think I need it to see what I need to tackle. I just pushed a button with a very long text in case you want to check it out.

Reply:
Well if this works I cant think of anything then :) You good to go. If something pops up we fix it then.